### DISCUSSION TITLE
Spatie Media Library always sets model_id as 0

### DISCUSSION THREAD

**Discussion by AleksandarV993** [spatie/laravel-medialibrary#3080] [Q&A]
Hey there, I'm using a custom column instead of a default `$table->id()`:
` $table->bigInteger("id")->unique();`
And whatever I do, and no matter how many new entries I add, the `model_id` column in the `media` table is always set to 0. Is there a way for me to change this? I'd like the media table to reference my main table.

**Comment by erikn69** [comment]
https://github.com/spatie/laravel-medialibrary/blob/ff4a546bafc0ecb4c779055fa1e5850c3b8b2bfe/database/migrations/create_media_table.php.stub#L12
`bigInteger` has no autoincrement, if you change that you must set it manually
